temporalio/worker/workflow_instance.py

```diff
@@ -0,0 +1,1299 @@
+            try:
+                success = await self._inbound.handle_query(input)
+                result_payloads = self._payload_converter.to_payloads([success])
+                if len(result_payloads) != 1:
+                    raise ValueError(
+                        f"Expected 1 result payload, got {len(result_payloads)}"
+                    )
+                command.respond_to_query.succeeded.response.CopyFrom(
+                    temporalio.bridge.worker.to_bridge_payload(result_payloads[0])
+                )
+            except Exception as err:
+                try:
+                    temporalio.exceptions.apply_exception_to_failure(
+                        err,
+                        self._payload_converter,
+                        command.respond_to_query.failed,
+                    )
+                except Exception as inner_err:
+                    raise ValueError(
+                        "Failed converting application error"
+                    ) from inner_err
+            finally:
+                self._pending_commands.append(command)
+
+        # Just find the arg types for now. The interceptor will be responsible
+        # for checking whether the query definition actually exists.
+        query_defn = self._queries.get(job.query_type)
+        arg_types, _ = (
+            self._type_lookup.get_type_hints(query_defn.fn)
+            if query_defn
+            else (None, None)
+        )
+        args = self._convert_payloads(job.arguments, arg_types)
+
+        # Schedule it
+        self.create_task(
+            run_query(
+                HandleQueryInput(
+                    id=job.query_id,
+                    name=job.query_type,
+                    args=args,
+                )
+            )
+        )
+
+    def _apply_resolve_activity(
+        self, job: temporalio.bridge.proto.workflow_activation.ResolveActivity
+    ) -> None:
+        handle = self._pending_activities.get(job.seq)
+        if not handle:
+            raise RuntimeError(f"Failed finding activity handle for sequence {job.seq}")
+        if job.result.HasField("completed"):
+            self._pending_activities.pop(job.seq)
+            ret: Optional[Any] = None
+            if job.result.completed.HasField("result"):
+                ret_types = [handle._input.ret_type] if handle._input.ret_type else None
+                ret_vals = self._convert_payloads(
+                    [job.result.completed.result],
+                    ret_types,
+                )
+                ret = ret_vals[0]
+            handle._resolve_success(ret)
+        elif job.result.HasField("failed"):
+            self._pending_activities.pop(job.seq)
+            handle._resolve_failure(
+                temporalio.exceptions.failure_to_error(
+                    job.result.failed.failure, self._payload_converter
+                )
+            )
+        elif job.result.HasField("cancelled"):
```

Comment:
I wish they were identical, but this references `job.result.cancelled.failure` whereas the above references `job.result.failed.failure`. At first glance it might seem like w/ such a small difference, the other 4 lines of code could be reused, but this is really 2 lines of code that the overly-wrapping formatter we use expands to 5.